src/dmd/backend/dvarstats.d

```diff
@@ -121,7 +119,7 @@ private extern (C) static int cmpLifeTime(scope const void* p1, scope const void
 }
 
 // a parent scope contains the creation offset of the child scope
-private static SYMIDX isParentScope(LifeTime* lifetimes, SYMIDX parent, SYMIDX si)
```

Comment:
Why did you remove the `static`?

Reply:
Oh,  thought this was at module level because it is not indented....

Reply:
Reverted